### DISCUSSION TITLE
Unofficial libspotify update. Will it ever happen?

### DISCUSSION THREAD

**Discussion by OneBobone** [librespot-org/librespot#624] [General]
Sorry if I am in the wrong forum, but I believe this group is the most comptent for the question above. I like many, have a player depending on libspotify that has lost key functionality.
Do you think that it is foreseeable that someone writes a “libspotify wrapper” around librespot and makes it happen?
It would be very valuable for many!

https://github.com/plietar/librespot/issues/209

**Comment by OneBobone** [comment]
Anyone have some smart insight?
:-)

**Comment by kingosticks** [comment]
Yes, it would be a good idea. If nothing else it would breathe new life into pyspotify. I'd love to do it if I had the time, which sadly I don't right now.
